#src/index.html

```diff
@@ -59,66 +59,79 @@
+                                <!-- <select name="date" id="date" class="form__select">
                                     <option value="date" class="form__option">Date</option>
-                                    <option value="" class="form__option"></option>
-                                    <option value="" class="form__option"></option>
-                                    <option value="" class="form__option"></option>
-                                </select>
+                                    <option value="2020.18.10" class="form__option">2020.18.10</option>
+                                    <option value="2020.07.20" class="form__option">2020.07.20</option>
+                                    <option value="2020.06.22" class="form__option">2020.06.22</option>
+                                </select>-->
+                                <input type="date" id="date" name="date" value="Date" class="form__select">
                             </div>
                         </form>
                         <button class="hero__button button col-md-4">Explore</button>
                     </div>
                     <div class="offset-md-1 col-md-5">
-                        <img src="img/thousand-01.png" alt="explore" class="hero__drawing-1 drawing-1">
+                        <img src="img/thousand-01.png" alt="explore" class="hero__illustration">
                     </div>
                 </div>
             </div>
         </div>
-        <div class="main__new-way new-way col-md-12">
-            <div class="new-way__container container">
-                <div class="new-way__row row">
+        <div class="main__article article col-md-12">
+            <div class="article__container container">
+                <div class="article__row row">
                     <div class="col-md-6">
-                        <img src="img/Thousand-02.png" alt="drawing-2" class="new-way__drawing-2 drawing-2">
+                        <img src="img/Thousand-02.png" alt="illustration" class="article__illustration">
                     </div>
-                    <div class="new-way__info info offset-md-1 col-md-5">
+                    <div class="article__info info offset-md-1 col-md-5">
                         <h2 class="info__title title">A new way to explore the<br>world</h2>
                         <p class="info__text text col-md-12">
                             For decades travellers have reached for Lonely Planet
                             books when looking to plan and execute their perfect 
                             trip, but now, they can also let Lonely Planet Experiences
                             lead the way
                         </p>
-                        <button class="info__button button col-md-4">Learn more</button>
+                        <a class="info__link">
+                            <button class="info__link-wrapper button col-md-4">
```

Comment:
no button should be nested inside an `a`
```suggestion
                            <a class="....... button col-md-4">Learn More</a>
```

no need for button tag, just apply button styles to a link with `button` class